terratag_test.go

```diff
@@ -153,19 +153,28 @@ func testTerraformWithFilter(t *testing.T, version string, filter string) {
 	}
 }
 
-func itShouldGenerateExpectedTerratagFiles(suiteDir string, g *GomegaWithT) {
-	expectedPattern := suiteDir + "/expected/**/*.terratag.tf"
+func itShouldGenerateExpectedTerratagFiles(entryDir string, g *GomegaWithT) {
+	expectedPattern := strings.Split(entryDir, "/out/")[0] + "/expected/*.tf"
 	var expectedTerratag []string
 	var actualTerratag []string
 	expectedTerratag, _ = doublestar.Glob(expectedPattern)
-	actualTerratag, _ = doublestar.Glob(suiteDir + "/out/**/*.terratag.tf")
+	if len(expectedTerratag) == 0 {
+		expectedPattern = strings.Split(entryDir, "/out/")[0] + "/expected/**/*.tf"
+		expectedTerratag, _ = doublestar.Glob(expectedPattern)
+	}
+	actualTerratag, _ = doublestar.Glob(entryDir + "/*.tf")
+	if len(actualTerratag) == 0 {
+		actualTerratag, _ = doublestar.Glob(entryDir + "/**/*.tf")
+	}
 	actualTerratag = filterSymlink(actualTerratag)
 
+	g.Expect(len(actualTerratag)).Should(BeNumerically(">", 0))
```

Comment:
these two g.Expect verify that we found the files. Before it was 0. Making sure it's larger than 0 - otherwise fail.